src/providers/authenticationProvider.ts

```diff
@@ -91,7 +91,9 @@ class TerraformCloudSessionHandler {
   }
 }
 export class TerraformCloudAuthenticationProvider implements vscode.AuthenticationProvider, vscode.Disposable {
-  static providerLabel = 'HashiCorp Terraform Cloud';
+  static providerLabel = 'HashiCorp Cloud Platform Terraform';
+  // These are IDs and session keys that are used to identify the provider and the session in VS Code secret storage
+  // we cannot change these in the rebrand without the user losing the previous session
   static providerID = 'HashiCorpTerraformCloud';
   private sessionKey = 'HashiCorpTerraformCloudSession';
```

Comment:
As far as I can tell, changing these will result in the user having to login again.

Reply:
Seems fine to me, as this is not user facing :)